azext_edge/edge/providers/orchestration/resources/schema_registries.py

```diff
@@ -66,7 +67,11 @@ def create(
         custom_role_id: Optional[str] = None,
         **kwargs,
     ) -> dict:
+        from ..rp_namespace import register_providers, ADR_PROVIDER
         with console.status("Working...") as c:
+            # Register providers
```

Comment:
not sure if we need this comment since the method name speaks for itself, or do you mean `# Register providers for schema registry`?

Reply:
I like how the comment is a different color from the code. That way the register providers is more visible and it wont be something paymaun finds 10 years later